docs/eigenda/overview.md

```diff
@@ -5,8 +5,8 @@ title: EigenDA Overview
 
 EigenDA is a data availability store made by
 [EigenLabs](https://www.eigenlayer.xyz/about) and built on top of
-[EigenLayer](../eigenlayer/overview/README.md), live on Goerli
-[testnet](https://blobs-goerli.eigenda.xyz/) and launching on mainnet in early
+[EigenLayer](../eigenlayer/overview/README.md), currently in transition from the
+Goerli testnet to the Holesky testnet and launching on mainnet in early
 Q2 2024.
```

Comment:
I wonder if it's worth even mentioning Goerli at this point since it's now completely deprecated?

Reply:
+1 - see my review comment on this section